core/field_dropdown.ts

```diff
@@ -453,17 +388,11 @@ export class FieldDropdown extends Field {
    * @param opt_newValue The input value.
    * @returns A valid language-neutral option, or null if invalid.
    */
-  protected override doClassValidation_(opt_newValue?: AnyDuringMigration):
-      string|null {
-    let isValueValid = false;
+  protected override doClassValidation_(opt_newValue?: MenuOption[1]): string
+      |null {
     const options = this.getOptions(true);
-    for (let i = 0, option; option = options[i]; i++) {
-      // Options are tuples of human-readable text and language-neutral values.
-      if (option[1] === opt_newValue) {
-        isValueValid = true;
-        break;
-      }
-    }
+    const isValueValid = options.some((option) => option[1] === opt_newValue);
```

Comment:
Yes, love this change!

Reply:
Condensing code's incredibly satisfying. Glad you like the change! ⭐